models/articles.js

```diff
@@ -26,7 +26,7 @@ exports.updateVotesByArticleId = (articleId, inc_votes) => {
     )
     .then((results) => {
       if (results.rows.length === 0) {
-        return Promise.reject({ status: 404, msg: "article not found" });
+        return Promise.reject({ status: 404, msg: "article not found" });git
```

Comment:
Think this 'git' at the end of the line might have been added my mistake?

Reply:
removed!